cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -112,13 +112,13 @@ public void revoke(HttpRequest httpRequest, HttpResponder httpResponder) throws
     verifyAuthRequest(request);
 
     // enforce that the user revoking access has admin privileges on the entity
-    authorizerInstantiator.get().enforce(request.getEntity(), SecurityRequestContext.toPrincipal(),
-                                                Action.ADMIN);
+    authorizer.enforce(request.getEntity(), SecurityRequestContext.toPrincipal(),
+                             Action.ADMIN);
```

Comment:
Actually, I can't do that until https://issues.cask.co/browse/CDAP-6603 is fixed (cannot get the AuthorizationCLITest to succeed without that change). Is it ok if I do that after CDAP-6603 is fixed?